AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -239,6 +249,19 @@ - (void)dealloc
   super.dataSource = nil;
 }
 
+/**
+ * A layout inspector implementation specific for the sizing behavior of UICollectionViewFlowLayouts
+ */
+- (ASCollectionViewFlowLayoutInspector *)flowLayoutInspector
+{
+    if (_flowLayoutInspector == nil) {
+        UICollectionViewFlowLayout *layout = (UICollectionViewFlowLayout *)self.collectionViewLayout;
```

Comment:
Please add an assertion checking the class type / that this cast is valid.

 (don't want runtime cost, do want immediate development safety-checking even internal to the framework)